src/prefect/tasks.py

```diff
@@ -973,6 +1002,7 @@ def task(
     refresh_cache: Optional[bool] = None,
     on_completion: Optional[List[Callable[["Task", TaskRun, State], None]]] = None,
     on_failure: Optional[List[Callable[["Task", TaskRun, State], None]]] = None,
+    viz_return_value: Any = None,
```

Comment:
Looks like this parameter needs to be added to the `task` overload above this one.

Reply:
thanks Alex! ✔️